### ISSUE TITLE
Alter Env Variable warning to be correct

### ISSUE THREAD

**Issue by minibikini** [ityonemo/librarian#39]
👋 Hi @ityonemo,

I've tried to pass env vars to SSH.run and it returned `:failure` and logged this:

```
[warn] this is currently unsupported until ERL-1107 is resolved
```

[ERL-1107](https://bugs.erlang.org/browse/ERL-1107) was resolved as _not a bug_. Does it mean the bug is in OpenSSH? Any workarounds?

**Comment by ityonemo** [comment]
oops, I need to remove that warning (it was my misunderstanding of SSH).  You need to set your ssh agent to allow the particular environment variables you're trying to send.  

https://stackoverflow.com/questions/4409951/can-i-forward-env-variables-over-ssh

**Comment by ityonemo** [comment]
changes completed in https://github.com/ityonemo/librarian/commit/5a7000eedcf4cabf2ef9467c97fb161027700830 and now available as 0.1.11 at hex.pm

**Comment by minibikini** [comment]
Thanks!
